src/components/ProgressRing.js

```diff
@@ -7,9 +7,10 @@ export class ProgressRing extends Component {
   }
 
   render() {
-    const { radius, stroke, progress, confirmationNumber } = this.props
+    const { radius, stroke, progress, confirmationNumber, hideConfirmationNumber } = this.props
     const { circumference, normalizedRadius } = this.state
     const strokeDashoffset = circumference - progress / 100 * circumference
+    const confirmations = hideConfirmationNumber ? '' : `${confirmationNumber}/8`
```

Comment:
This is an improvement for the user experience. Right now when you make a transfer, you see the message `Waiting for Block Confirmations...` and inside the spinner you can see how blocks confirmation are being incremented `0/8`, `2/8`...`8/8`. When `8/8` block of confirmation are reached, text changes to `Validators Verifying Transaction...` and the spinner is almost completed. The problem was that the numbers of blocks `8/8` was still being displayed even after we finished waiting for block confirmations, so users were confused because they see `8/8`  but the spinner and loading process were not completed. So now the `8/8` is removed when we finish waiting for blocks confirmations.

Before
![screenshot_20181010_155245](https://user-images.githubusercontent.com/4614574/46759049-bb11aa00-cca4-11e8-8d2c-3f3d1213fd32.png)

After
![screenshot_20181010_155335](https://user-images.githubusercontent.com/4614574/46759058-bf3dc780-cca4-11e8-9c70-6d5de41bb020.png)

Reply:
Is this related to https://github.com/poanetwork/bridge-ui/issues/119?
Will it wait for 8 block confirmations even if the number configured in contract is less?

Reply:
This is not related to #119. Loading will wait for 8 block confirmations as it was, it doesn't check for number configured in contract.

Reply:
Ok. I think later we need to return the counter (`0/n`, `1/n`, `n/n`) when we address #119. My view that it is useful to see what exactly we are waiting for.

Reply:
Just to be clear the counter (`n/8`) will continue to be displayed but only when `Waiting for Block Confirmations...` message is being displayed

Reply:
Thanks for clarification! So, forget of my previous comment.